Traffic_flow.ipynb

```diff
@@ -141,8 +166,8 @@
     "    plt.show()\n",
     "    \n",
     "interact(jam,\n",
-    "         rho_l=FloatSlider(min=0.,max=0.9,value=0.2,description=r'$\\rho_l$'),\n",
-    "         t=FloatSlider(min=0.,max=1.,value=0.2));"
+    "         rho_l=widgets.FloatSlider(min=0.1,max=0.9,value=0.2,description=r'$\\rho_l$'),\n",
```

Comment:
This is reverting changes that I made in the bookform PR.  Was this intentional?  I suspect not.  I'm very worried now that much of my work will be lost unless we go over this PR with a fine tooth comb...